frontend/src/app/Settings/Facility/Components/OrderingProvider.tsx

```diff
@@ -105,66 +105,94 @@ const OrderingProvider: React.FC<Props> = ({
           }
           name="NPI"
           required={isRequired}
-          value={provider.NPI || ""}
-          onChange={onChange}
-          onBlur={() => {
-            validateField("orderingProvider.NPI");
-          }}
+          value={formCurrentValues.orderingProvider.NPI ?? undefined}
+          registrationProps={register("orderingProvider.NPI", {
+            required: isRequired
+              ? "NPI should be a 10-digit numerical value (##########)"
+              : "",
+            validate: {
+              validNPI: (npi: string) =>
+                npi?.length
+                  ? isValidNPI(npi) ||
```

Comment:
could we pull this string into some sort of shared variable with the above? don't want a typo / string drift to result in this check not returning the right value